### PR TITLE
Calc em and threshold

### PR DESCRIPTION
**Pull request by sgoycoechea**

### CODE REVIEW

File: scripts/calculate-threshold-and-em/calculate_em.py [hunk 1]
```diff
@@ -0,0 +1,41 @@
+import os
```
Comment (bryant1410) [inline scripts/calculate-threshold-and-em/calculate_em.py:2]: Remember to set the execution permissions to this file and the other ones. Also the shebang line (and to the other files):

```suggestion
#!/usr/bin/env python
import os
```

File: scripts/calculate-threshold-and-em/process_answer_indexes.py [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+    sentences_untokenized.close()
+    sentences_tokenized.close()
+    answer_indexes_untokenized.close()
+    answer_indexes_tokenized.close()
+
+
+def main() -> None:
+    process_answer_indexes("./sentences.es", "./sentences.32k.es", "./answers_indexes.es", "./answers.es")
+    process_answer_indexes("./sentences.en", "./sentences.32k.en", "./answers_indexes.en", "./answers.en")
```
Comment (bryant1410) [inline scripts/calculate-threshold-and-em/process_answer_indexes.py]: The "./" aren't necessary!

File: scripts/generate-alignments/generate_alignments.sh [hunk 3]
```diff
@@ -12,7 +12,7 @@ spm_encode --model=es.model --output_format=piece < sentences.es > sentences.32k
 
 ../scripts/generate-alignments/process_answer_indexes.py
 
-../thualign/bin/generate.sh -s spanish -o output.txt
+../thualign/bin/generate.sh -s spanish -o output
```
Comment (bryant1410) [inline scripts/generate-alignments/generate_alignments.sh:15]: In other parts of the code, there's the file `output.txt`. Is this fine?

File: scripts/generate-alignments/generate_alignments.sh [hunk 4]
```diff
@@ -12,7 +12,7 @@ spm_encode --model=es.model --output_format=piece < sentences.es > sentences.32k
```
Comment (bryant1410) [inline scripts/generate-alignments/generate_alignments.sh:12]: Don't make this file non-executable!

**Review by bryant1410 — CHANGES_REQUESTED** [review]

File: scripts/generate-alignments/generate_alignments.sh (hunk 3 above)
Comment (sgoycoechea) [reply scripts/generate-alignments/generate_alignments.sh:15]: Yes, the extension is appended afterwards.

**Review by bryant1410 — APPROVED** [review]
I'm gonna fix the executable bit on the file after merging

**Pull request merged**